### PR TITLE
docs: add a user guide for ensuring network connectivity for clones

### PR DESCRIPTION
**Pull request by raduiliescu**
## Reason for This PR

We need a user guide or recommendations on how to setup the network for clones.

## Description of Changes

Adding a document presenting the strategy to ensure continued network connectivity for multiple clones created from a single Firecracker microVM snapshot.

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided (issue no. or explanation).
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.

### CODE REVIEW

**Review by berciuliviu — DISMISSED** [review]
LGTM

**Review by AlexandruCihodaru — APPROVED** [review]
🚀

File: docs/snapshotting/snapshot-support.md [hunk 1]
```diff
@@ -87,7 +88,8 @@ target is under **8ms** with 5ms p90 for a microvm with this specs:
 
 - High snapshot latency on 5.4+ host kernels - 
 [#2129](https://github.com/firecracker-microvm/firecracker/issues/2129)
-- Guest network connectivity is not guaranteed to be preserved after resume
+- Guest network connectivity is not guaranteed to be preserved after resume.
+For ensuring guest network connectivity please see [Network connectivity for clones](network-for-clones.md)
```
Comment (acatangiu) [inline docs/snapshotting/snapshot-support.md:92]: I would make it clear that active network connections cannot be guaranteed (even with the doc):

```suggestion
- Guest network connectivity is not guaranteed to be preserved after resume.
For ensuring guest network connectivity please see [Network connectivity for clones](network-for-clones.md). Active network connections are not guaranteed to survive a snapshot cycle.
```

Comment (raduiliescu) [reply docs/snapshotting/snapshot-support.md]: I modified the wording. PTAL

**Review by berciuliviu — APPROVED** [review]

**Review by dianpopa — APPROVED** [review]

**Pull request merged**